corehq/apps/domain/tests/test_domain_calculated_properties.py

```diff
@@ -7,75 +7,48 @@
 from corehq.apps.es.tests.utils import es_test
 from corehq.apps.sms.models import INCOMING, OUTGOING
 from dimagi.utils.parsing import json_format_datetime
-from pillowtop.es_utils import initialize_index_and_mapping
 
 from corehq.apps.domain.calculations import all_domain_stats, calced_props, sms, get_sms_count
 from corehq.apps.domain.models import Domain
-from corehq.elastic import get_es_new, refresh_elasticsearch_index
-from corehq.pillows.mappings.case_mapping import CASE_INDEX_INFO
-from corehq.pillows.mappings.sms_mapping import SMS_INDEX_INFO
-from corehq.pillows.mappings.xform_mapping import XFORM_INDEX_INFO
-from corehq.pillows.mappings.user_mapping import USER_INDEX_INFO
-from corehq.util.elastic import ensure_index_deleted
-from corehq.elastic import send_to_elasticsearch
+from corehq.apps.es.cases import case_adapter
+from corehq.apps.es.forms import form_adapter
+from corehq.apps.es.sms import sms_adapter
+from corehq.apps.es.users import user_adapter
 
 
-@es_test
+@es_test(requires=[case_adapter, form_adapter, sms_adapter, user_adapter])
 class BaseCalculatedPropertiesTest(TestCase):
+
     @classmethod
     def setUpClass(cls):
         super(BaseCalculatedPropertiesTest, cls).setUpClass()
-        cls.es = [{
-            'info': index_info,
-            'instance': get_es_new(),
-        } for index_info in [CASE_INDEX_INFO, SMS_INDEX_INFO, XFORM_INDEX_INFO, USER_INDEX_INFO]]
-
         cls.domain = Domain(name='test-b9289e19d819')
         cls.domain.save()
 
     @classmethod
     def tearDownClass(cls):
         cls.domain.delete()
```

Comment:
nit: this could be converted to use `addClassCleanup` and `tearDownClass` could be eliminated

Reply:
See dacf2c78670